operator-framework-core/src/main/java/io/javaoperatorsdk/operator/api/RegisteredController.java

```diff
@@ -0,0 +1,22 @@
+package io.javaoperatorsdk.operator.api;
+
+import java.util.Set;
+
+import io.javaoperatorsdk.operator.processing.Controller;
+
+public class RegisteredController {
+
+  private Controller<?> controller;
+
+  public RegisteredController(Controller<?> controller) {
```

Comment:
Putting this class in the same package as `Operator` would allow us to make this constructor package-only.

Reply:
Moved to the same package. Also changed it to an interface, that is implemented by the controller. What do you think?